modules/ossm-configuring-distr-tracing-tempo.adoc

```diff
@@ -66,35 +139,45 @@ kind: Kiali
 spec:
   external_services:
     tracing:
-      query_timeout: 30
+      query_timeout: 30 #<1>
       enabled: true
       in_cluster_url: 'http://tempo-sample-query-frontend.tracing-system.svc.cluster.local:16685'
       url: '[Tempo query frontend Route url]'
-      use_grpc: true # <1>
+      use_grpc: true # <2>
 ----
-<1> If you are not using the default HTTP or gRPC port for Jaeger or Tempo, replace the `in_cluster_url:` port with your custom port.
+<1> The default `query_timeout` is 30. If you set it greater than 30, you need to update `.spec.server.write_timeout` in the Kiali CR and add the annotation `haproxy.router.openshift.io/timeout=50s` to the Kiali route. Both `.spec.server.write_timeout` and `haproxy.router.openshift.io/timeout=` must be greater than `query_timeout`.
+<2> If you are not using the default HTTP or gRPC port for Jaeger or Tempo, replace the `in_cluster_url:` port with your custom port.
 +
 [NOTE]
 ====
 Kiali 1.73 uses the Jaeger Query API, which causes a longer response time depending on Tempo resource limits. If you see a `Could not fetch spans` error message in the Kiali UI, then check your Tempo configuration or reduce the limit per query in Kiali.
 ====
 
-. Create a TempoStack instance using the Red Hat {TempoOperator} in the `tracing-system` namespace. For more information, see "Installing the distributed tracing platform (Tempo)" in the "Additional resources" section.
+. Send requests to your application.
+
+. Verify the `istiod` pod logs and the `otel-collector` pod logs.
+
+== Configuring the {TempoName} in a mTLS encrypted Service Mesh member namespace
+
+[NOTE]
+====
+You don't need this additional DestinationRule configuration if you have created a TempoStack instance in a regular namespace i.e. not in a Service Mesh member namespace.
+====
+
+When you enable Service Mesh dataPlane mTLS encryption and you create a TempoStack instance in a Service Mesh member namespace such as `tracing-system-mtls`, all traffic is TLS encrypted. This is not expected from the Tempo distributed service and it returns TLS error in this scenario.
+
+You can apply the following example to disable the TLS trafficPolicy for fixing the {TempoName} TLS error.
```

Comment:
Don't we also need DestinationRule for Kiali as it is described in the Google Doc?

Reply:
yes, from Kiali configuration.